Froggie/Source/Froggie.Domain/Users/IGetUsersGroupsQuery.cs

```diff
@@ -0,0 +1,8 @@
+﻿using Froggie.Domain.Groups;
+
+namespace Froggie.Domain.Users;
+
+public interface IGetUsersGroupsQuery
+{
+    ValueTask<IReadOnlyCollection<Group>> QueryAsync(Id<User> userId);
```

Comment:
Can you explain why you do this again? (and by "this" I mean the ValueTask wrapper)